exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -467,75 +450,22 @@ impl fmt::Display for JavaArtifactId {
     }
 }
 
-#[derive(Serialize, Deserialize, Clone, ProtobufConvert, PartialEq)]
-#[protobuf_convert(source = "proto::ServiceStateHashes")]
-struct ServiceStateHashes {
-    instance_id: u32,
-    state_hashes: Vec<Vec<u8>>,
-}
-
-impl From<&ServiceStateHashes> for (InstanceId, Vec<Hash>) {
-    fn from(value: &ServiceStateHashes) -> Self {
-        let hashes: Vec<Hash> = value
-            .state_hashes
-            .iter()
-            .map(|bytes| to_hash(bytes))
-            .collect();
-        (value.instance_id, hashes)
-    }
-}
-
-#[derive(Serialize, Deserialize, Clone, ProtobufConvert, BinaryValue, PartialEq)]
-#[protobuf_convert(source = "proto::ServiceRuntimeStateHashes")]
-struct ServiceRuntimeStateHashes {
-    runtime_state_hashes: Vec<Vec<u8>>,
-    service_state_hashes: Vec<ServiceStateHashes>,
-}
-
-impl ServiceRuntimeStateHashes {
-    fn runtime(&self) -> Vec<Hash> {
-        self.runtime_state_hashes
-            .iter()
-            .map(|bytes| to_hash(bytes))
-            .collect()
-    }
-
-    fn instances(&self) -> Vec<(InstanceId, Vec<Hash>)> {
-        self.service_state_hashes
-            .iter()
-            .map(|service| service.into())
-            .collect()
-    }
-}
-
-impl From<ServiceRuntimeStateHashes> for StateHashAggregator {
-    fn from(value: ServiceRuntimeStateHashes) -> Self {
-        StateHashAggregator {
-            runtime: value.runtime(),
-            instances: value.instances(),
-        }
-    }
-}
-
-fn to_hash(bytes: &[u8]) -> Hash {
-    Hash::from_bytes(bytes.into()).unwrap()
-}
-
 type ExceptionHandler = Fn(&JNIEnv, JObject) -> ExecutionError;
 struct ExceptionHandlers;
 
 impl ExceptionHandlers {
     const DEFAULT: &'static ExceptionHandler = &|env, exception| {
         assert!(!exception.is_null(), "No exception thrown.");
         let message = describe_java_exception(env, exception);
-        (Error::JavaException, message).into()
+        // TODO(ECR-4015):
```

Comment:
It is closed as 'Won't do', it's not clear what needs to be done.

Reply:
Accidental artifact